core/embed/projects/prodtest/README.md

```
@@ -806,6 +806,15 @@ prodtest-mem-read
 OK 01020304
 ```
 
+### prodtest-uptime
+Returns the device uptime (time elapsed since the last boot) in milliseconds.
+
+Example:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)